starknet_devnet/starknet_wrapper.py

```diff
@@ -105,6 +106,8 @@ def __init__(self, config: DevnetConfig):
         self.fee_token = FeeToken(self)
         self.accounts = Accounts(self)
         self.__udc = UDC(self)
+        self.mempool = []
```

Comment:
If these properties are only meant to be accessed from inside starknet_wrapper, they can be prefixed with a double underscore (`__`).

Reply:
but is also used in `TransactionHandler` class

Reply:
It is sort of a private class, but if the linter complains, let's leave it as it is.

Reply:
yes I tried to add this but tests were failing and I realized that it's also used in `TransactionHandler`

Reply:
Fixed 7d8ed9a605d98923d6c801e996bcecf437bd17b5